pyaedt/application/Design.py

```diff
@@ -1,7 +1,7 @@
 """
 This module contains these classes: `Design` and `DesignCache`.
 
-It contains all basic project information and objects. 
+This module provides all functionalities for basic project information and objects. 
 Because this module is inherited in the Main Tool class, 
 it will be a simple call from it.
```

Comment:
```suggestion
These classes is are inherited in the main tool class.
```

Effectively, this means that

Reply:
Fix "is are" in Alex's suggestion.